packages/composite-checkout/src/components/checkout-payment-methods.tsx

```diff
@@ -23,6 +23,10 @@ const debug = debugFactory( 'composite-checkout:checkout-payment-methods' );
 
 const CheckoutPaymentMethodsWrapper = styled.div`
 	padding-top: 4px;
+	> div > div[disabled]:has( + div[disabled]:hover )::before,
+	> div > div[disabled]:has( + div.is-checked[disabled] )::before {
+		border-bottom: none;
+	}
```

Comment:
All this faffery is to ensure hover states look okay when the form is disabled. 🤷🏻 

Not really necessary, but it's a detail.